Sources/Commands/SwiftBuildTool.swift

```diff
@@ -213,14 +211,14 @@ public struct SwiftBuildTool {
         print("USAGE: swift build [mode] [options]")
         print("")
         print("MODES:")
-        print("  --configuration <value>        Build with configuration (debug|release) [-c]")
-        print("  --clean[=<mode>]               Delete artifacts (build|dist)")
+        print("  --configuration <value>   Build with configuration (debug|release)")
+        print("  --clean [<mode>]          Delete artifacts (build|dist)")
```

Comment:
Yes, you're right.  Good catch.  Technically we could still parse it if we treat any argument that starts with a dash as an option, but let's not go there, at least not now.  I've been sketching out a much more uniform option-handling machinery, and there will be time to discuss that in the reviews for that API when the time comes.